config_defaults_inc.php

```diff
@@ -851,13 +851,27 @@
 $g_window_title = 'MantisBT';
 
 /**
+ * OpenSearch engine title prefix.
+ * This is used to describe Browser Search entries, and must be short enough
+ * so that when inserted into the 'opensearch_XXX_short' language string, the
+ * resulting text is 16 characters or less, to be compliant with the limit for
+ * the ShortName element as defined in the OpenSearch specification.
+ * @link http://www.opensearch.org/Specifications/OpenSearch/1.1
+ * @see $g_window_title
+ * @global string $g_search_title
+ */
+$g_search_title = '%window_title%';
```

Comment:
> about 7% better performance in 2nd benchmark

Not bad indeed, I did not expect that much actually. I'm still concerned about the loss of "just in time" evaluation of configs however, especially for paths that are quite likely to have been changed by admins - we need to be very careful not to cause regressions. Consider this example:

_config_defaults_inc.php_
```
$g_path = (dynamically built from $_SERVER) // e.g. http://internal-hostname.example.com/mantisbt/
$g_icon_path = $g_path . 'images/';
```
_config_inc.php_
```
$g_path = 'http://example.com/mantisbt/'; # required e.g. due to use of reverse proxy
```

In this case, `$g_icon_path == 'http://internal-hostname.example.com/mantisbt/images/`, which is invalid in the reverse-proxy scenario; this would force the admin to manually define all variables referencing $g_path (formerly %path% in their config_inc.php (and knowing that they have to do so). Not very user-friendly, and likely to cause support issues when upgrading.

> Do you think it's worth to think more about it (obsolete cookie seetings, ...) 

Yes definitely. And get others' opinions too.

> target it for a version > 2.0?

For sure a 2.x target, if we decide to move forward.